scripts/prod_deploy/nginx.conf

```diff
@@ -0,0 +1,18 @@
+server {
+    listen 80;
+    server_name 54.174.175.215;
```

Comment:
To catch any hostnames or IP addresses this should be: 
```
server {
    listen       80  default_server;
    server_name  _;
```
http://nginx.org/en/docs/http/server_names.html